contracts/base/ModuleManager.sol

```diff
@@ -130,7 +131,8 @@ contract ModuleManager is SelfAuthorized, Executor {
     /// @dev Returns array of modules.
     /// @param start Start of the page.
     /// @param pageSize Maximum number of modules that should be returned.
-    /// @return Array of modules.
+    /// @return array Array of modules.
+    /// @return next ??.
```

Comment:
This is a paginated function, so `next` would be used as `start` for the next page

Reply:
Added a commit for this.